### PR TITLE
Return person data together with session recording

### PR DESCRIPTION
**Pull request by macobo**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: ee/clickhouse/queries/clickhouse_session_recording.py [hunk 1]
```diff
@@ -33,9 +33,11 @@
 
 
 class SessionRecording(BaseSessionRecording):
-    def query_recording_snapshots(self, team: Team, session_id: str) -> List[Any]:
+    def query_recording_snapshots(self, team: Team, session_id: str) -> Tuple[Optional[str], List[Any]]:
```
Comment (paolodamico) [inline ee/clickhouse/queries/clickhouse_session_recording.py:36]: Very minor thing, maybe worth adding a small comment clarifying that the returned tuple will contain the session's ID & the snapshot data?

**Review by paolodamico — APPROVED** [review]
lgtm!

**Comment by paolodamico** [comment]
I just realized, shouldn't we include these changes too in the Postgres side?

File: ee/clickhouse/queries/clickhouse_session_recording.py (hunk 1 above)
Comment (macobo) [reply ee/clickhouse/queries/clickhouse_session_recording.py]: Did one better - named the types :)

**Comment by macobo** [comment]
> I just realized, shouldn't we include these changes too in the Postgres side?

Both were changed (the clickhouse impl inherits from postgres one and they share tests).

**Pull request merged**